### ISSUE TITLE
Container.getAll does not return named bindings

### ISSUE THREAD

**Issue by alexmt** [inversify/InversifyJS#405]
Method Container.getAll does not return named bindings

## Expected Behavior
All bindings including named should be returned by method

## Current Behavior
Currently Container.getAll filters out named bindings. 

## Steps to Reproduce

Please use following example to reproduce issue:

Example:
```

let container = new Container();
container.bind(TYPE.Controller)
  .toConstantValue(new AppController()).whenTargetNamed('AppController');
container.getAll(TYPE.Controller);

```

```
node_modules/inversify/lib/planning/planner.js:97
            throw new Error(error.message);
            ^
Error: No bindings found for serviceIdentifier: Symbol(Controller)
```

## Context
I'm trying to use `inversify-restify-utils` library to integrate inversify and restify. Library uses named bindings and method Container.getAll to resolve controller bindings (https://github.com/inversify/inversify-restify-utils/blob/master/src/server.ts#L53 )

## Your Environment

- nodejs: 6.3.1
- inversify: 3.0.0-beta.1
- os: mac os 10.12.1

**Comment by remojansen** [comment]
Hi @alexmt please use inversify@2.0.1 for the moment. We are in the process of upgrading `inversify-restify-utils` to work with 3.0.0-beta.1 but is not ready yet.

**Comment by alexmt** [comment]
Thank you for response, @remojansen ! We are currently using 2.0.1 . I was trying to implement test app using 3.0.0-beta.1 in order to create PR for https://github.com/inversify/InversifyJS/issues/412 . I would be glad to do it when 3.0.0-beta.1 is stable.

**Comment by remojansen** [comment]
Done by #420 and available on [3.0.0-beta.2](https://github.com/inversify/InversifyJS/releases/tag/3.0.0-beta.2)
